mygithub/src/components/github/Profile.js

```diff
@@ -1,58 +1,58 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import RepoList from './RepoList.js';
-class Profile extends Component{
-	
-	render(){
-// Nesting profile component in App component
-		return(
+class Profile extends Component {
+
+	render() {
+		// Nesting profile component in App component
+		return (
 			<div>
 				<div className="panel panel-default">
-					  <div className="panel-heading"></div>
-					  <div className="panel-body">
-					    <h1> Name :{this.props.userData.name} </h1>
-					  </div>
+					<div className="panel-heading"></div>
+					<div className="panel-body">
+						<h1> Name :{this.props.userData.name} </h1>
```

Comment:
Minor suggestion: Consider adding a check for `this.props.userData` before accessing its properties to prevent potential errors if the data is not yet available. This can be done using optional chaining or a conditional statement.